src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -298,7 +283,27 @@ const AppWrapper = (props: { children: ReactNode }) => {
       await SecureStorage.delete(KeyStoreKeys.APP_OP_PASSWORD);
       await SecureStorage.delete(KeyStoreKeys.SIGNIFY_BRAN);
     }
+    await loadDatabase();
     await loadCacheBasicStorage();
+    const agentUrls = await Agent.agent.getAgentUrls();
+    if (agentUrls.url) {
+      try {
+        await Agent.agent.start(agentUrls.url);
+        setIsOnline(true);
+      } catch (e) {
+        const errorStack = (e as Error).stack as string;
+        // If the error is failed to fetch with signify, we retry until the connection is secured
+        if (/SignifyClient/gi.test(errorStack)) {
+          Agent.agent.connect().then(() => {
+            setIsOnline(Agent.agent.getKeriaOnlineStatus());
+          });
+        } else {
+          throw e;
+        }
+      }
+    } else {
+      // @TODO: on the onboarding screen to enter it
```

Comment:
We need to cover this case now or develop will break - can you check with Duke on how to proceed here?

Reply:
@iFergal After discussing with Duke, I will merge this PR to Duke's branch

Reply:
Shall we merge to feature/DTIS-895-secure-seed-phrase-generation-and-verify-process branch (Duke is doing on this)?

Reply:
OK once Duke has that branch we can merge to it